packages/edit-site/src/components/header/index.js

```diff
@@ -35,27 +34,15 @@ export default function Header( {
 	isNavigationOpen,
 	onToggleNavigation,
 } ) {
-	const {
-		deviceType,
-		hasFixedToolbar,
-		template,
-		page,
-		showOnFront,
-	} = useSelect( ( select ) => {
+	const { deviceType, hasFixedToolbar, template } = useSelect( ( select ) => {
 		const {
 			__experimentalGetPreviewDeviceType,
 			isFeatureActive,
 			getTemplateId,
 			getTemplatePartId,
 			getTemplateType,
-			getPage,
 		} = select( 'core/edit-site' );
-
-		const { getEntityRecord, getEditedEntityRecord } = select( 'core' );
-		const { show_on_front: _showOnFront } = getEditedEntityRecord(
-			'root',
-			'site'
-		);
```

Comment:
I've just noticed that we are losing all the "show on front" logic from the page switcher. 🤔 

https://github.com/WordPress/gutenberg/blob/c1208188447b78e2b4e983c8b8f21a9f6c0d27f9/packages/edit-site/src/components/page-switcher/index.js#L69-L88

Am I missing something?

Reply:
Oooops 😄 Pushed a commit which re-adds it.